scripts/error-codes/__tests__/dev-expression-with-codes-test.js

```diff
@@ -25,18 +25,17 @@ function compare(input, output) {
 
 var oldEnv;
 
-describe('dev-expression', function() {
```

Comment:
Append ?w=1 to the pull request and it should hide them from the diff, at least :)